package.json

```diff
@@ -86,7 +89,10 @@
       "@typescript-eslint/explicit-function-return-type": "off",
       "@typescript-eslint/no-empty-interface": "warn",
       "@typescript-eslint/explicit-module-boundary-types": "off",
-      "@typescript-eslint/no-explicit-any": "warn"
+      "@typescript-eslint/no-explicit-any": "warn",
+      "prettier/prettier": "warn",
```

Comment:
The linter needs to error if the code style rules are not followed. See https://and-murray.slack.com/archives/C04S1EX1JVC/p1697623746932599

Reply:
For this one you could add `"plugin:prettier/recommended"` into the eslint "extends" section to replace the existing extends/plugin/rules that are applied for prettier.  See https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

It was only necessary to manually add the below because we were going for warn rather than error. 

```
    "prettier/prettier": "warn",
    "arrow-body-style": "off",
   "prefer-arrow-callback": "off"
```

Reply:
For this one you could add `"plugin:prettier/recommended"` into the eslint "extends" section to replace the existing extends/plugin/rules that are applied for prettier.  See https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

It was only necessary to manually add the below because we were going for warn rather than error. 

```
    "prettier/prettier": "warn",
    "arrow-body-style": "off",
   "prefer-arrow-callback": "off"
```